grugru-default.el

```diff
@@ -49,7 +49,35 @@ with `grugru-remove-on-major-mode' or `grugru-remove-global'."
     (symbol "setq" "setq-default")
     (word   "global" "local"))
    ((tex-mode latex-mode yatex-mode)
-    (symbol "figure" "table"))))
+    (symbol "figure" "table"))
+   ((org-mode) ;; v9.3.6
```

Comment:
```suggestion
   (org-mode ;; v9.3.6
```
Parenthesis seems to be unnecessary, because these words will be rotated only in org-mode, even in the future.